src/components/table.js

```diff
@@ -450,7 +484,21 @@ export const GridTable = Utils.forwardRefWithName('GridTable', ({
           containerRole: 'presentation',
           'aria-readonly': null, // Clear out ARIA properties which have been moved one level up
           'aria-label': `${tableName} content`, // The whole table is a tab stop so it needs a label
-          noContentRenderer: () => div({ style: { marginTop: '1rem', textAlign: 'center', fontStyle: 'italic' } }, [noContentMessage]),
+          noContentRenderer: () => {
+            return div({
```

Comment:
This exact div formatting and styling seems oft repeated. Can we abstract that out to a component? Just noting that the changes are repeated three times, and I could see that being easy to overlook in the future.

Reply:
Yep, that's a great point! Actually, now that I look at it some other parts of the table structure could probably be extracted out too, but I'll at least do this one. :)